mlflow/sklearn.py

```diff
@@ -424,3 +424,29 @@ def load_model(model_uri):
     return _load_model_from_local_file(
         path=sklearn_model_artifacts_path, serialization_format=serialization_format
     )
+
+
+class _SklearnTrainingSession(object):
```

Comment:
Can we document the class constructor with information about the `clazz` and `allow_children` parameters & also document the `should_log()` method with information about the behavior?